addons/mass_mailing/static/src/xml/mass_mailing.xml

```diff
@@ -15,4 +15,18 @@
             </a>
         </t>
     </div>
+    <t t-extend="ListView.buttons" t-name="MailingContactListView.buttons">
```

Comment:
Shoudln't those be in their specific files (either one for all "backend views" or one for each view ?) Just to avoid having "a bit of everything-related xml".